app/views/api/guidance/index.html.erb

```diff
@@ -1,11 +1,139 @@
+        <h2 class="govuk-heading-m">
+          What's new
+        </h2>
 
-<p>
-  <%= link_to "test page (nested)", api_guidance_page_path(page: 'nested/nested/test') %>
-</p>
+        <p class="govuk-body">
+          <%= @latest_release_note.date %>: <%= @latest_release_note.content %>
+        </p>
+
+        <p class="govuk-body">
+          <%= govuk_link_to("Read the full release notes for more details.", "https://github.com/DFE-Digital/npq-registration/blob/main/release-notes.md") %>
+        </p>
+        <p class="govuk-body">
+          <%= govuk_link_to("Sign up to get update emails about the NPQs data management API for lead providers.", "#") %>
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full-from-desktop govuk-!-padding-top-8">
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <h2 class="govuk-heading-l govuk-!-font-weight-bold">What you can do in the API</h2>
+        <p class="govuk-body">Guidance on how lead providers can integrate with this API so they can view, submit and
+          update data. The data enables providers to manage and deliver NPQ-based training, and ensure they get paid the
+          right amount by the DfE</p>
+        <p class="govuk-body govuk-!-margin-bottom-0">
+          <%= govuk_link_to "Browse guidance", api_guidance_path, class: "govuk-!-font-weight-bold" %></p>
+      </div>
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <h2 class="govuk-heading-l">Definitions</h2>
+        <p class="govuk-body">Take a look at the key concepts and data states we use</p>
+        <p class="govuk-body govuk-!-margin-bottom-0">
+          <%= govuk_link_to "Browse definitions", api_guidance_page_path(page: 'definitions'), class: "govuk-!-font-weight-bold" %></p>
+      </div>
+      <div class="govuk-grid-column-one-third-from-desktop">
+        <h2 class="govuk-heading-l">API latest version</h2>
+        <p class="govuk-body">Access the v3.0.0 of the NPQs data management API for lead providers</p>
+        <p class="govuk-body">
+          <%= govuk_link_to "Browse v.3.0.0 of the API", api_guidance_page_path(page: 'api-latest-version'), class: "govuk-!-font-weight-bold" %></p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full-from-desktop">
+      <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+      <div class="govuk-grid-column-two-thirds ">
+        <h2 class="govuk-heading-l">API previous versions </h2>
+        <p class="govuk-body">
+          <%= govuk_link_to("View API v1.0.0", "#", class: "govuk-!-font-weight-bold") %>
+        </p>
+        <p class="govuk-body">
+          <%= govuk_link_to("View API v2.0.0", "#", class: "govuk-!-font-weight-bold") %>
+        </p>
```

Comment:
I know this is just placeholder for now, but semantically this should be a list.

```suggestion
        <ul class="govuk-list">
          <li><%= govuk_link_to("View API v1.0.0", "#", class: "govuk-!-font-weight-bold") %></li>
          <li><%= govuk_link_to("View API v2.0.0", "#", class: "govuk-!-font-weight-bold") %></li>
        </ul>
```